public/utils/constants.ts

```diff
@@ -59,6 +59,9 @@ export const AD_DOCS_LINK =
 export const AD_HIGH_CARDINALITY_LINK =
   'https://opensearch.org/docs/latest/observing-your-data/ad/index/#optional-set-category-fields-for-high-cardinality';
 
+export const AD_FEATURE_ANYWHERE_LINK =
```

Comment:
this link just sends to the main opensearch.org doc page

Reply:
I think that's because the doc side pr hasn’t merged yet. So the link doesn't exist yet

Reply:
but it will be the link for feature anywhere documentation page